### PR TITLE
ui/15399700-sub navigation

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

Screenshot 2023-04-21 at 08 34 04

### CODE REVIEW

**Review by krawczykkatarzyna — APPROVED** [review]

File: src/components/common/Extensions/SubNavigation/Partials/DropdownContent/DecideDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { LEARN_MORE_ADMIN } from '~constants';
+import Icon from '~shared/Icon/Icon';
+
+import LinkItem from '../LinkItem';
+import LearnMore from '../LearnMore';
+
+import { MSG } from './consts';
+import styles from './DropdownContent.module.css';
+
+const displayName = 'common.Extensions.SubNavigation.Partials.DropdownContent.DecideDropdown';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/SubNavigation/Partials/DropdownContent/DecideDropdown.tsx:14]: `partials` should be lowercase (directory and displayName)

File: src/components/common/Extensions/SubNavigation/Partials/DropdownContent/DecideDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const DecideDropdown: React.FC<PropsWithChildren> = () => (
+  <div>
+    <div className="text-gray-400 text-xs">
+      <div className="hidden md:block mx-4 mt-4">
+        <FormattedMessage {...MSG.decisionsContentTitle} />
+      </div>
+      <ul className={styles.listWrapper}>
+        <LinkItem title={MSG.createDecision} description={MSG.createDecisionDescription} />
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/SubNavigation/Partials/DropdownContent/DecideDropdown.tsx:23]: Maybe we should `react-intl` here?

File: src/components/common/Extensions/SubNavigation/Partials/DropdownContent/DropdownContent.module.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+.infoWrapper {
+  @apply mt-6 md:mt-8 mb-4 text-sm flex justify-center font-normal items-center;
+}
+
+.questionIcon {
+  @apply mr-2;
+
+  & svg {
+    display: block;
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/SubNavigation/Partials/DropdownContent/DropdownContent.module.css:25]: use `@apply`

File: src/components/common/Extensions/SubNavigation/Partials/DropdownContent/ManageDropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+const displayName = 'common.Extensions.SubNavigation.Partials.DropdownContent.ManageDropdown';
+
+const ManageDropdown: React.FC<PropsWithChildren> = () => (
+  <div>
+    <div className="text-gray-400 text-xs">
+      <div className="hidden md:block mx-4 mt-4">
+        <FormattedMessage {...MSG.manageContentTitle} />
+      </div>
+      <ul className={styles.listWrapper}>
+        <LinkItem title={MSG.manageTeams} description={MSG.manageTeamsDescription} />
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/SubNavigation/Partials/DropdownContent/ManageDropdown.tsx:23]: Maybe we should `react-intl` here?

File: src/components/common/Extensions/SubNavigation/Partials/DropdownContent/PayDropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+const displayName = 'common.Extensions.SubNavigation.Partials.DropdownContent.PayDropdown';
+
+const PayDropdown: React.FC<PropsWithChildren> = () => (
+  <div>
+    <div className="text-gray-400 text-xs">
+      <div className="hidden md:block mx-4 mt-4">
+        <FormattedMessage {...MSG.paymentContentTitle} />
+      </div>
+      <ul className={styles.listWrapper}>
+        <LinkItem title={MSG.singlePayments} description={MSG.singlePaymentsDescription} />
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/SubNavigation/Partials/DropdownContent/PayDropdown.tsx:24]: Maybe we should `react-intl `here?

File: src/components/common/Extensions/SubNavigation/SubNavigationMobile.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+import { useDetectClickOutside } from '~hooks';
+
+import SubNavigationItemMobile from './SubNavigationItem/SubNavigationItemMobile';
+import { subNavigationItems } from './consts';
+import styles from './SubNavigationMobile.module.css';
+
+const displayName = 'Extensions.SubNavigation.SubNavigationMobile';
+
+const SubNavigationMobile: React.FC<PropsWithChildren> = () => {
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/SubNavigation/SubNavigationMobile.tsx:11]: Use only `FC` here and everywhere

**Pull request merged**